unity-ggjj/Assets/Scripts/Input/UnityInputSystemInterface.cs

```diff
@@ -74,7 +74,11 @@ void Controls.IUIActions.OnLeftMouseButton(InputAction.CallbackContext context)
     /// <param name="context"></param>
     void Controls.IUIActions.OnDirectionalButtons(InputAction.CallbackContext context)
     {
-        //Unused for now
+        var value = context.ReadValue<Vector2>();
+        if (context.performed && value != Vector2.zero)
+        {
+            _onDirectionalButtons.Invoke(value);
+        }
```

Comment:
Nit: I'm sure you already know what's coming at this point. : D
```suggestion
        if (!context.performed)
        {
            return;
        }
        
        if (value == Vector2.zero)
        {
            return;
        }
        _onDirectionalButtons.Invoke(value);
```

Reply:
Usually I'd agree, but I think in this case it needs to be specifically context.performed, as "!context.performed" could mean context.started and context.cancelled are true which would changed the functionality. 

Saying that I realised I should use context.started instead of checking if the direction is the zero vector.